doc/api/process.md

```diff
@@ -1282,7 +1282,12 @@ Will generate:
 
 `heapTotal` and `heapUsed` refer to V8's memory usage.
 `external` refers to the memory usage of C++ objects bound to JavaScript
-objects managed by V8.
+objects managed by V8. `rss`, Resident Set Size, is the total space allocated in
+memory for the process, which includes the _heap_, _code segment_ and _stack_.
+
```

Comment:
A little more clarity on `rss` will help people disambiguate it from the virtual memory size.
For example: 
`rss`, Resident Set Size, is the amount of space occupied in the main memory device (that is a subset of the total allocated memory) for the process, which includes the _heap_, _code segment_ and _stack_.

Reply:
Makes sense, thanks. Fixing 👍